src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -235,17 +235,71 @@ const myFade = {
 <header transition:animate={myFade}> ... </header>
 ```
 
-## Preventing client-side navigation
+## Router control
 
-Once you have enabled client-side routing on a page by adding the `<ViewTransitions />` component, every anchor on the page that links to another page on your site will be navigated via client-side routing. There are some cases where you might want to not navigate via CSR. One example is if the link is to any non-page, such as a PDF within your `public/` folder. Or an API route that produces an image.
+The `<ViewTransitions />` router handles navigation by listening to:
 
-In these cases, you can opt out of client-side routing on a per-link basis using the `data-astro-reload` attribute like so:
+- Clicks on `<a>` elements.
+- Backwards and forwards navigation events.
 
-```html
-<a href="/quarterly-earnings.pdf" data-astro-reload>
+The following sections discuss how you can further control when navigation occurs within the router.
```

Comment:
```suggestion
The following options allow you to further control when navigation occurs within the router:

- `astro-data-reload`: an `<a>` tag attribute to force a full-page navigation
- `astro-data-history="auto | push | replace"`: an `<a>` tag attribute to control the browser's history
- `navigate(href, options)`: a method available to any client script or client component to trigger navigation
```
I think this is the introduction to the section I was wanting! Now, the sub sections below can be more usage based and we don't as easily lose these options inside specific user activities.